ksqldb-engine/src/main/java/io/confluent/ksql/analyzer/Analyzer.java

```diff
@@ -658,16 +658,6 @@ public Void visitFunctionCall(final FunctionCall functionCall, final Void contex
     }
 
     public void validate() {
-
-      if (containsUserColumnsWithSameNameAsNewPseudoColumns(analysis, ksqlConfig)) {
```

Comment:
How come previously we were passing in a separate ksqlConfig here but in the new version of the code we're using the one in the analysis? Are these always the same? (Didn't get a chance to check myself -- forgive my laziness on this one.)

Reply:
Yes, they're always the same AFAICT. Was just a mistake I made earlier to include the config.